Source/platform/ps4/sys/endian.h

```diff
@@ -0,0 +1,10 @@
+//
+// Created by cpasjuste on 18/04/22.
+//
+
+#ifndef DEVILUTIONX_ENDIAN_H
+#define DEVILUTIONX_ENDIAN_H
+
+#include <endian.h>
```

Comment:
Perhaps this should simply be fixed in https://github.com/diasurgical/libmpq?

Reply:
OrbisOS is based on freebsd, which normally includes <machine/endian.h>. This is also where the official sony sdk puts this file. OpenOrbis uses a modified version of musl, which assumes a linux kernel (and thus puts endian.h directly under /usr/include). This issue will occur on many other projects too, so its best fixed in upstream musl repo if you ask me.

Reply:
Hi guy,

I'm not planning to support the PS4 port, I just helped a bit on request :) Feel free to do any modifications !

Reply:
@Cpasjuste, it is much appreciated! I added my comments to inform reviewers of the situations so they can make an informed decision about the PR. Another note worth mentioning is that progress on openorbis-musl is somewhat sporadic, so perhaps it makes sense to merge this.

Reply:
Yeah no problem mate :)
They just need to know that I won't be able to put more time in it anytime soon. But yes:
- libmpq should be patched (and "platform/ps4" removed), I didn't notice it was a "devix" repo.
- some should verify if pkg modules are really needed (it's possible that they aren't needed anymore since I made some changes to the PS4 "pacbrew" toolchain package manager.